pkg/specgen/generate/container_create.go

```diff
@@ -21,6 +21,27 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func platformToCrunVariant(arch string, os string) string {
```

Comment:
We can move this `platformToCrunVariant` and `modifyCrunIfNeeded` to elsewhere like `c/common` but other than `podman` nothing else is using it.

Reply:
Where would user customize these settings?  What happens if a new WASM OCI runtime shows up.
Should be called platformToOCIRuntime()

Reply:
* `platformToOCIRuntime(` sounds better.
* I think in future there should be a configurable map in `containers.conf` which can allow users to map various OCI runtimes to various `arch` and `os`.

Reply:
Yes or to turn it off.

Reply:
Open a PR to add a table, for name to OCIRuntime